src/components/PlayerControls.tsx

```diff
@@ -45,20 +45,20 @@ const PlayerControls: FunctionComponent<PlayerControlsProps> = ({
             )}
           </div>
           <div className="flex items-center">
-            {/* <ListboxInput value={'' + speed} onChange={setSpeed}>
+            <ListboxInput value={speed} onChange={setSpeed}>
```

Comment:
I typed everything above instead of this "casting" hack. I'd recommend using `speed as string` in this circumstance for casting

https://basarat.gitbook.io/typescript/type-system/type-assertion